Adding a CCECE 2025 paper section to the News and Status Reports page

pamoroso opened the pull request:
This change adds to the [News and Status Reports](https://interlisp.org/project/status) page a section on the CCEECE 2025 talk by Eleanor Young with a link to the preprint. It addresses [issue #2175](https://github.com/Interlisp/medley/issues/2175).

pamoroso committed:
Initial commit

Upload IEEE CCEECE 2025 paper preprint to link from the News and Status Reports page.

Signed-off-by: Paolo Amoroso <paolo.amoroso@gmail.com>

pamoroso committed:
Add CCECE 2025 section

Add a section on the CCEECE 2025 talk by Eleanor Young with a link to the preprint. This addresses Medley issue #2175.

Signed-off-by: Paolo Amoroso <info@paoloamoroso.com>

stumbo commented on content/en/project/status/_index.md:
```diff
@@ -22,6 +22,12 @@ The [2024 Medley Interlisp Annual Report](/project/status/2024medleyannualreport
 
+## May 2025 IEEE Canadian Conference on Electrical and Computer Engineering
+
+On May 28, 2025 Eleanor Young [gave the talk](https://ccece2025.ieee.ca/technical-program) "The Medley Interlisp Project: Reviving a Historical Software System" at the [2025 IEEE Canadian Conference on Electrical and Computer Engineering and Industry Summit](https://ccece2025.ieee.ca) (CCECE). The paper by Young et al. was accepted for publication.
```
How about replacing 'gave the talk' with 'presented'?  I would also drop the link to the technical program.  That's a rat hole that we don't need to be directing people to.  They can get to it from the conference link, if interested.

pamoroso replied:
I applied these changes and the ones in your other comment, thanks.

stumbo commented on content/en/project/status/_index.md:
```diff
@@ -22,6 +22,12 @@ The [2024 Medley Interlisp Annual Report](/project/status/2024medleyannualreport
+
+On May 28, 2025 Eleanor Young [gave the talk](https://ccece2025.ieee.ca/technical-program) "The Medley Interlisp Project: Reviving a Historical Software System" at the [2025 IEEE Canadian Conference on Electrical and Computer Engineering and Industry Summit](https://ccece2025.ieee.ca) (CCECE). The paper by Young et al. was accepted for publication.
+
+You can [download the preprint of the paper](/documentation/young-ccece2025.pdf). For questions and inquiries about the paper please post to a [communication channel we monitor](../getInvolved#3-join-in-the-discussions).
```
Maybe change the sentence to something like this:

A [preprint]() of the paper is available.  Please post questions, inquiries or your thoughts on the paper to one of our [social media sites]().

pamoroso committed:
Rephrase talk description/preprint, drop CCECE link

Rephrase the reference to the CCECE talk. Remove the link to the CCECE technical program. Rephrase the reference to the downloadable preprint for smoother flow.

These changes address feedback by Bill Stumbo.

Signed-off-by: Paolo Amoroso <info@paoloamoroso.com>

masinter reviewed (commented):
lgtm
